samples/django/README.md

```diff
@@ -0,0 +1,17 @@
+# Django Samples
+
+These two sample Django apps provide a skeleton for the two main use cases of the
+Django contrib helpers
```

Comment:
Missing period on this line (it looks funny when I read it [rendered](https://github.com/waprin/oauth2client/tree/e13c1f2f6f9d59746a537ae8439bd1e99d49494e/samples/django)).